eng/pipelines/coreclr/templates/xplat-job.yml

```diff
@@ -72,21 +72,6 @@ jobs:
     - name: buildConfig
       value: ${{ parameters.buildConfig }}
 
-    - ${{ if eq(parameters.buildConfig, 'checked') }}:
```

Comment:
NIT: Mono template is also defining this, so now you moved it up to xplat-setup so you can remove from there as well.

Reply:
Weird I thought I included this in my change as well. Will do that, thanks for catching it.